watched_keywords.txt

```diff
@@ -811,7 +811,7 @@
 1504789259	tripleee	corintdesign\.tumblr\.com
 1504847330	tripleee	hypoclense
 1504854896	tripleee	wewilltransportit\.com
-1504855612	tripleee	surveymonkey\.com(?<!developer\.surveymonkey\.com)
+1504855612	tripleee	surveymonkey\.com(?=/\w)(?<!developer\.surveymonkey\.com)(?<!api\.surveymonkey\.com)(?<!help\.surveymonkey\.com)(?<!widget\.surveymonkey\.com)(?!(?:[^<]|<(?!\/?code>))*+<\/code>)
```

Comment:
These can be refactored to `(?<!(?:api|developer|help|survey)\.surveymonkey\.com)`

Reply:
That results in a non-fixed-length lookbehind, which is incompatible with MS search.